README.md

```diff
@@ -1,6 +1,6 @@
 # Dart samples
 
-[![Build Status](https://travis-ci.org/dart-lang/samples.svg?branch=master)](https://travis-ci.org/dart-lang/samples)
+![example workflow name](https://github.com/dart-lang/samples/workflows/Dart%20CI/badge.svg)
```

Comment:
Gah! one more thing! Make this link to the action?

Reply:
The best link I could find to use was https://github.com/google/mono_repo.dart/actions?query=branch%3Amaster  - the [actions page](https://github.com/google/mono_repo.dart/actions) uses search syntax 🤷

Reply:
LGTM!